packages/engine/Source/Scene/Scene.js

```diff
@@ -1881,8 +1881,8 @@ Scene.prototype.updateFrameState = function () {
   frameState.globeTranslucencyState = this._globeTranslucencyState;
 
   if (defined(this.globe)) {
-    frameState.terrainExaggeration = this.globe.terrainExaggeration;
-    frameState.terrainExaggerationRelativeHeight = this.globe.terrainExaggerationRelativeHeight;
+    frameState.verticalExaggeration = this.globe.terrainExaggeration;
+    frameState.verticalExaggerationRelativeHeight = this.globe.terrainExaggerationRelativeHeight;
```

Comment:
I was going to point out that vertical exaggeration for 3D Tiles only works if the scene has a globe, but I think that's covered by this task:

> Update public API. Exaggeration is still controlled via `globe.terrainExaggeration` and `globe.terrainExaggerationRelativeHeight`. This should really be changed to `scene.verticalExaggeration` and `scene.verticalExaggerationRelativeHeight`.